packages/components/src/components/icon/icons.js

```diff
@@ -181,6 +181,7 @@ import './common/ic-bank.svg';
 import './common/ic-bell.svg';
 import './common/ic-black-warning.svg';
 import './common/ic-block.svg';
+import './common/ic-blocked-advertisers-barred.svg';
```

Comment:
icon was not pushed in previous PR. when running npm run build, this icon will always be auto generated